operations/node_management/Rebuild_NCNs.md

```diff
@@ -4,18 +4,34 @@ Rebuild a master, worker, or storage non-compute node (NCN). Use this procedure
 
 The following is a high-level overview of the NCN rebuild workflow:
 1. [Prepare Node](#prepare_nodes)
-   * There is a different procedure for each type of node (worker, master, and storage).
+    * There is a different procedure for each type of node (worker, master, and storage).
 1. [Identify Node and Update Metadata](#identify_node_and_update_metadata)
-   * Same procedure for all node types.
-1. [Rebuild Node](#rebuild_node)
-   * Same for master and worker nodes, but different for storage nodes.
+    * Same procedure for all node types.
+1. [Wipe Disks](#wipe_disks)
+    * Same for master and worker nodes, but different for storage nodes.
+1. [Power Cycle Node](#power_cycle_node)
+    * Same procedure for all node types.
+1. [Rebuild Storage Node](#rebuild__storage_node)
+    * Only needed for storage nodes
 1. [Validation](#validation)
-   * There is a different procedure for each type of node (worker, master, and storage).
+    * There is a different procedure for each type of node (worker, master, and storage).
 
 ## Prerequisites
 
 The system is fully installed and has transitioned off of the LiveCD.
 
+<a name=set-var></a>
+For several of the commands in this section, you will need to have variables set with the name of the node being rebuilt and its xname.
```

Comment:
Most procedures with multiple prereqs have them listed in an unordered list. It might be helpful to do this here, indenting the commands and context for this prereq.